ansible/roles/zuul/files/layout.yaml

```diff
@@ -355,6 +355,10 @@ projects:
   - name: Juniper/vijava
     template:
       - name: ci-contrail-windows-other-projects-template
+      
+    - name: Juniper/contrail-go-api
```

Comment:
Indentation level seems to be inconsistent.

Reply:
Fixed